test/verify/check-client

```diff
@@ -39,7 +39,8 @@ class TestClient(testlib.MachineCase):
         if self.m_target.image.startswith("debian") or self.m_target.image.startswith("ubuntu"):
             self.m_target.execute("dpkg --purge cockpit-pcp-dbgsym || true; dpkg --purge cockpit-pcp pcp")
         elif self.m_target.image != 'arch':
-            self.m_target.execute("rpm --erase --verbose cockpit-pcp pcp; systemctl daemon-reload")
+            # TODO: remove conditional when all images have python3-pcp and a Python PCP bridge
+            self.m_target.execute("rpm -q python3-pcp && rpm --erase --verbose cockpit-pcp pcp python3-pcp || rpm --erase --verbose cockpit-pcp pcp fi; systemctl daemon-reload")
```

Comment:
I buy a `;` :grin:

Reply:
Also, please use `if` instead of `&& ||`. That matches the intended `fi` and is more robust.